build/Build.cs

```diff
@@ -47,6 +43,10 @@ partial class Build : NukeBuild
     [GitRepository] readonly GitRepository GitRepository;
     [GitVersion(Framework = "net6.0")] readonly GitVersion GitVersion;
 
+    [Parameter] string NugetPublishUrl = "https://api.nuget.org/v3/index.json";
```

Comment:
Every `[Parameter]` can be overridden by passing the value when executing `build.cmd`